src/tools/rbd_ggate/main.cc

```diff
@@ -405,23 +404,6 @@ int main(int argc, const char *argv[]) {
     exit(0);
   }
 
-  std::string conf_file_list;
-  std::string cluster;
-  CephInitParameters iparams = ceph_argparse_early_args(
-      args, CEPH_ENTITY_TYPE_CLIENT, &cluster, &conf_file_list);
-
-  md_config_t config;
-  config.name = iparams.name;
-  config.cluster = cluster;
-
-  if (!conf_file_list.empty()) {
-    config.parse_config_files(conf_file_list.c_str(), nullptr, 0);
-  } else {
-    config.parse_config_files(nullptr, nullptr, 0);
-  }
-  config.parse_env();
-  config.parse_argv(args);
-
```

Comment:
I suppose whole this section could be replaced with
```
// filter out ceph config options
md_config_t().parse_argv(args);
```
But we can't just remove it.